src/cdk/stepper/stepper.ts

```diff
@@ -166,12 +170,12 @@ export class CdkStep implements OnChanges {
   /** Whether step is marked as completed. */
   @Input()
   get completed(): boolean {
-    return this._customCompleted == null ? this._getDefaultCompleted() : this._customCompleted;
+    return this._completedOverride == null ? this._getDefaultCompleted() : this._completedOverride;
   }
   set completed(value: boolean) {
-    this._customCompleted = coerceBooleanProperty(value);
+    this._completedOverride = coerceBooleanProperty(value);
   }
-  private _customCompleted: boolean | null = null;
+  _completedOverride: boolean|null = null;
```

Comment:
is there a reason this was made non-private?

Reply:
It's been a long time since I opened this PR, but from what I can tell, it's because `CdkStepper` also checks it.

Reply:
Ah ok, yeah sorry was going through my old assigned PRs and found this that I never responded to